.github/workflows/pull-request-schedule/trigger-builds.js

```diff
@@ -0,0 +1,65 @@
+module.exports = async ({ github, context }) => {
+    // PullRequestQuery is a GraphQL query to find pull requests with the proper
+    // label
+    const PullRequestQuery = `
+        query($owner: String!, $repo: String!, $label: String!) {
+            repository(owner: $owner, name: $repo) {
+                labels(first: 10, query: $label) {
+                    nodes {
+                        name
+                        pullRequests(states: OPEN, last: 10) {
```

Comment:
Does this `last: 10` in the query for OPEN pullRequests means to obtain only the last 10 PRs? Or is it a pagination thing or the like?

Reply:
You're right, this means we only get 10 PRs (with the label); that's obviously wrong (I forgot to change the testing values I used).  I should double check what order this is in (by testing with a different label), and see if picking the _first_ PRs would make sense; since we will eventually remove the labels (when the PRs get scheduled) that should work fine.

Reply:
Fixed in a40333443489f8407f4b019b8e1b25669dcc0250